docs/applications/big-data/big-data-in-the-linode-cloud-streaming-data-processing-with-apache-storm.md

```diff
@@ -175,57 +172,57 @@ You can set up multiple Cluster Manager nodes if necessary to satisfy organizati
         ~~~
 
     Restart the SSH service to enable this change:
-            
+
         service ssh restart
 
     {: .caution}
     >
     > Since access to the cluster manager provides access to all Storm and Zookeeper clusters and any sensitive data they are processing, its security configuration should be considered critical, and access should be as restrictive as possible.
 
-9. Log in to the Cluster Manager Node as its `root` user, using the public IP address that was shown when you created the node:
+9.  Continue from this same Linode to log in to the Cluster Manager Node as the `root` user, using the public IP address that was shown when you created the node:
```

Comment:
This log in is done from the user's workstation to Cluster manager linode, not from some other linode to cluster manager linode. So, I feel "continue from this same linode" is incorrect.